.circleci/config.yml

```diff
@@ -403,3 +453,32 @@ workflows:
           filters:
             branches:
               only: /^build-pr-.*/
+
+      - build-android-unsigned:
+          context: mattermost-mobile-unsigned
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
+            branches:
+              ignore: /.*/
```

Comment:
isn't this ignoring every branch?

I'm not really sure which are the conditions for a unsigned release

Reply:
This every branch is being omitted and only executing when a tag is added, this only to be generated for the **final** build.